### PR TITLE
Minor miscellaneous fixes for v4.4

### PR DESCRIPTION
**Pull request by pazner**
Some very minor fixes and changes… (targeting the `mfem-4.4-dev` branch).

### CODE REVIEW

File: doc/makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ DOXYGEN_CONF = CodeDocumentation.conf
 # doxygen uses: graphviz, latex
 html: $(DOXYGEN_CONF)
 	@# Generate the html documentation
-	@doxygen $(DOXYGEN_CONF)
+	@( cat $(DOXYGEN_CONF) ; echo "$(MFEM_DOXY_FLAGS)" ) | doxygen -
```
Comment (pazner) [inline doc/makefile:19]: The use case here is something like
```
make MFEM_DOXY_FLAGS="HAVE_DOT = NO"
```
(dot really slows down the documentation generation).

File: fem/pfespace.cpp [hunk 2]
```diff
@@ -3541,7 +3541,6 @@ DeviceConformingProlongationOperator::DeviceConformingProlongationOperator(
    const int tdofs = R->Height();
    MFEM_ASSERT(tdofs == R->HostReadI()[tdofs], "");
    ltdof_ldof = Array<int>(const_cast<int*>(R->HostReadJ()), tdofs);
-   ltdof_ldof.UseDevice();
```
Comment (pazner) [inline fem/pfespace.cpp:3544]: This is a no-op. More generally, `UseDevice()` is a bit of a footgun, it's so similar to `UseDevice(true)`

File: tests/scripts/branch-history [hunk 3]
```diff
@@ -149,7 +149,7 @@ foreach my $sha (@commits) {
         # File was added
         $blob_size += int(`git cat-file -s $dst`); }
       elsif ($mode eq "DD") { }
-      elsif ($mode eq "MM") {
+      elsif ($mode eq "MM" || $mode eq "MR") {
```
Comment (pazner) [inline tests/scripts/branch-history:152]: I was getting the error "Unknown git status letter" on a different branch without this change

**Review by tzanio — APPROVED** [review]

File: doc/makefile (hunk 1 above)
Comment (v-dobrev) [reply doc/makefile:19]: I see this slowdown on Mac with recent versions of `graphviz/dot` as well. It used to be much faster. Also, in github actions the build is relatively fast (1m 1s), so `Ubuntu 18.04.6 LTS` does not seem affected by this slowdown yet.

**Review by v-dobrev — APPROVED** [review]

**Pull request merged**